docs/book/orders.rst

```diff
@@ -4,117 +4,108 @@
 Orders
 ======
 
-*Order* model is one of the most important in Sylius, it represents the order placed via your store! It has a very consistent and clear API, which allows you to easily manipulate and process orders.
+**Order** model is one of the most important in Sylius, where many concepts of e-commerce meet.
+It represents an order that can be either placed or in progress.
 
-Customer Reference
-------------------
+**Order** holds a collection of **OrderItem** instances, which represent products from the shop,
+as it physical copies, with chosen variants and quantities.
 
-*Order* holds a reference to specific *User*, which is available through ``getUser()`` method:
+The Order is **assigned to the channel** in which it has been created. Moreover the **language** the customer was using
+and the **currency with its exchange rate** at the moment of creation are saved.
+
+How to create an Order?
+-----------------------
+
+To programatically create an Order you will of course need a factory.
 
 .. code-block:: php
 
-    echo $order->getUser()->getEmail(); // john@example.com
+    /** @var FactoryInterface $order */
+    $orderFactory = $this->get('sylius.factory.order');
 
-When creating order programatically, you can define the user yourself:
+    /** @var OrderInterface $order */
+    $order = $orderFactory->createNew();
+
+But before it can be saved using the repository it needs to have a **Currency**.
+You can get it from the currency context of your application, which will provide you with its code.
+By the way you can add also the exchange rate of your currency.
 
 .. code-block:: php
 
-    $order = $this->get('sylius.repository.order')->createNew();
-    $john = $this->get('sylius.repository.user')->find(3);
+    /** @var string $currencyCode */
+    $currencyCode = $this->container->get('sylius.context.currency')->getCurrency()->getCode();
+
+    $order->setCurrencyCode($currencyCode);
 
-    $order->setUser($john);
+    /** @var RepositoryInterface $currencyRepository */
+    $currencyRepository = $this->get('sylius.repository.currency');
 
-*Order* may not have reference to *User* in case when *Order* was created by guest.
+    $currency = $currencyRepository->findOneByCode($currencyCode);
```

Comment:
Isn't it a bit confusing for some people to get the Currency object again through the repository, since you already have it of the context?